spec/acceptance/default_parameters_spec.rb

```diff
@@ -32,15 +32,22 @@ class { 'postgresql::globals':
         version             => '9.3',
       }->
       class { 'postgresql::server': } ->
-      deploy::file { 'jdk-7u71-linux-x64.tar.gz':
-        target          => $jh,
-        fetch_options   => '-q -c --header "Cookie: oraclelicense=accept-securebackup-cookie"',
-        url             => #{java_url},
-        download_timout => 1800,
-        strip           => true,
+      file { $jh:
+        ensure => 'directory',
+      } ->
+      archive { '/tmp/jdk-7u71-linux-x64.tar.gz':
+        ensure          => present,
+        extract         => true,
+        extract_command => 'tar xfz %s --strip-components=1',
+        extract_path    => $jh,
+        source          => "#{java_url}",
+        creates         => "${jh}/bin",
+        cleanup         => true,
+        cookie          => 'oraclelicense=accept-securebackup-cookie',
       } ->
       class { 'jira':
         version      => '6.2.7',
+        checksum     => 'ddacb69b9327bd40ab6ce726f0dd4a08',
```

Comment:
I would like to change the `download_url` from `http` to `https`, even if Atlassian redirects the traffic and provide the `checksum` for our `version`. This way the module should work out of the box. 

But I do not have enough time to update the Atlassian modules to the latest version. This would require more testing regarding the facts and that Confluence 5.8 dropped the support of Java 7 etc.